usage attribution logs spans (#14975)
DataDog/documentation: content/en/account_management/billing/usage_attribution.md

{{< img src="account_management/billing/usage_attribution/Usage-Attribution-v2-Total-Usage.png" alt="Applied tags in Datadog" style="width:100%;" >}}

> - Data is shown by all tag keys selected e.g. by app and service.

Reviewer: ```suggestion
- Data is shown by all tag keys selected. For example, by app and service.
```

Daily data can also be pulled using the tool's public API. For more information, see the [API endpoint documentation][2].

> 

Reviewer: ```suggestion
[2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution
```
Moving this link back. Unfortunately, our pre-commit hook currently breaks these links by moving them down in the document. I have a PR up to fix it.

{{< partial name="whats-next/whats-next.html" >}}

[1]: https://docs.datadoghq.com/api/v1/usage-metering/#get-monthly-usage-attribution

> [2]: https://docs.datadoghq.com/api/v1/usage-metering/#get-hourly-usage-attribution

Reviewer: ```suggestion
```
